Simplenote/TagListViewController.swift

```diff
@@ -331,6 +322,11 @@ extension TagListViewController: UITableViewDataSource, UITableViewDelegate {
     func tableView(_ tableView: UITableView, editingStyleForRowAt indexPath: IndexPath) -> UITableViewCell.EditingStyle {
         return .none
     }
+
+    func reloadTableView() {
+        tableView.reloadData()
+        updateCurrentSelection()
```

Comment:
Why do we need to "recalculate" the selected row? is it to deal with selectedRowIndexPath changes applied via the new UITableView extension?

Reply:
We update selected index path in case the currently selected row changed its index path after the reload. Can happen if we add, delete or reorder tags.

Reply:
Thanks for the clarification!